### PR TITLE
Logging Z-Stream Release Notes correction - 4.8 branch

### PR DESCRIPTION
**Pull request by libander**
Version(s): 4.8

Issue: https://issues.redhat.com/browse/RHDEVDOCS-4780

Link to docs preview:

QE review:
- [x] QE has approved this change.

Additional information: Roll up of z-stream release notes already approved and live on 4.9+. This corrects release note cherry pick failures to the 4.8 branch.

### CODE REVIEW

**Comment by libander** [comment]
- https://github.com/openshift/openshift-docs/pull/53584
- https://github.com/openshift/openshift-docs/pull/53576
- https://github.com/openshift/openshift-docs/pull/52842
- https://github.com/openshift/openshift-docs/pull/52530

File: logging/cluster-logging-release-notes.adoc [hunk 1]
```diff
@@ -1,6 +1,8 @@
-[id="cluster-logging-release-notes"]
-= Release notes for Red Hat OpenShift Logging 5.4
+:_content-type: ASSEMBLY
+[id="cluster-logging-release-notes_{context}"]
```
Comment (rolfedh) [inline logging/cluster-logging-release-notes.adoc:2]:
```suggestion
[id="cluster-logging-release-notes"]
```
Assembly files don't have a parent document from which to inherit a context value. I also spot-checked a few assemblies to make sure they didn't have this context attribute: they don't.

File: logging/cluster-logging-release-notes.adoc [hunk 2]
```diff
@@ -372,21 +386,25 @@ This release includes link:https://access.redhat.com/errata/RHSA-2021:5129[RHSA-
 
 [id="cluster-logging-release-notes-5-3-0_{context}"]
 == OpenShift Logging 5.3.0
-The following advisories are available for OpenShift Logging 5.3.x:
+This release includes link:https://access.redhat.com/errata/RHSA-2021:4627[RHSA-2021:4627 OpenShift Logging Bug Fix Release 5.3.0]
 
 [id="openshift-logging-5-3-0-new-features-and-enhancements_{context}"]
 === New features and enhancements
-* With this update, authorization requirements for Log Forwarding have been relaxed. Outputs may now be configured with SASL, username/password, or TLS.
+* With this update, authorization options for Log Forwarding have been expanded. Outputs may now be configured with SASL, username/password, or TLS.
```
Comment (rolfedh) [inline logging/cluster-logging-release-notes.adoc:393]:
```suggestion
* With this update, authorization options for Log Forwarding have been expanded. Outputs can now be configured with SASL, username/password, or TLS.
```

File: logging/cluster-logging-release-notes.adoc [hunk 3]
```diff
@@ -372,21 +386,25 @@ This release includes link:https://access.redhat.com/errata/RHSA-2021:5129[RHSA-
...
 
 [id="openshift-logging-5-3-0-bug-fixes_{context}"]
 === Bug fixes
+* Before this update, if you forwarded logs using the syslog protocol, serializing a ruby hash encoded key/value pairs to contain a '=>' character and replaced tabs with "#11". This update fixes the issue so that log messages are correctly serialized as valid JSON. (link:https://issues.redhat.com/browse/LOG-1494[LOG-1494])
```
Comment (rolfedh) [inline logging/cluster-logging-release-notes.adoc:397]: The verb tenses and commas in this sentence make it confusing. Can you rephrase this, please?

Comment (rolfedh) [reply logging/cluster-logging-release-notes.adoc:397]: `Ruby hash-encoded key-value pairs`

File: logging/cluster-logging-release-notes.adoc [hunk 4]
```diff
@@ -907,23 +922,23 @@ To see these metrics, open the *Administrator* perspective in the {product-title
...
 
-* Before this update, the AWS CloudWatch Fluentd plugin logged its AWS API calls to the Fluentd log at all log levels, consuming additional {product-title} node resources. With this update, the AWS CloudWatch Fluentd plugin logs AWS API calls only at the "debug" and "trace" log levels. This way, at the default "warn" log level, Fluentd does not consume extra node resources. (link:https://issues.redhat.com/browse/LOG-1071[LOG-1071])
+* Before this update, the AWS CloudWatch Fluentd plug-in logged its AWS API calls to the Fluentd log at all log levels, consuming additional {product-title} node resources. With this update, the AWS CloudWatch Fluentd plug-in logs AWS API calls only at the "debug" and "trace" log levels. This way, at the default "warn" log level, Fluentd does not consume extra node resources. (link:https://issues.redhat.com/browse/LOG-1071[LOG-1071])
```
Comment (rolfedh) [inline logging/cluster-logging-release-notes.adoc:933]: This [plugin guideline](https://redhat-documentation.github.io/supplementary-style-guide/#plugin) changed recently. Please replace `plug-in` with `plugin`.

File: modules/cluster-logging-rn-5.3.13.adoc [hunk 5]
```diff
@@ -0,0 +1,36 @@
+//module included in cluster-logging-release-notes.adoc
+:_content-type: REFERENCE
+[id="cluster-logging-release-notes-5-3-13_{context}"]
+= Logging 5.3.13
+This release includes link:https://access.redhat.com/errata/RHSA-2022:6882[RHSA-2022:68828-OpenShift Logging Bug Fix Release 5.3.13].
+
+[id="openshift-logging-5-3-13-bug-fixes"]
+== Bug fixes
+* Before this update, the log file size map generated by the `log-file-metrics-exporter` component did not remove entries for deleted files, resulting in increased file size, and process memory. With this update, the log file size map does not contain entries for deleted files. (link:https://issues.redhat.com/browse/LOG-3293[LOG-3293])
```
Comment (rolfedh) [inline modules/cluster-logging-rn-5.3.13.adoc:9]:
```suggestion
* Before this update, the log file size map generated by the `log-file-metrics-exporter` component did not remove entries for deleted files, resulting in increased file size and process memory. With this update, the log file size map does not contain entries for deleted files. (link:https://issues.redhat.com/browse/LOG-3293[LOG-3293])
```

File: modules/cluster-logging-rn-5.4.9.adoc [hunk 6]
```diff
@@ -0,0 +1,85 @@
+//module included in cluster-logging-release-notes.adoc
+:_content-type: REFERENCE
+[id="cluster-logging-release-notes-5-4-9_{context}"]
+= Logging 5.4.9
+This release includes link:https://access.redhat.com/errata/RHBA-2022:8780[OpenShift Logging Bug Fix Release 5.4.9].
+
+[id="openshift-logging-5-4-9-bug-fixes"]
+== Bug fixes
+* Before this update, the Fluentd collector would warn of unused configuration parameters. This update removes those configuration parameters and their warning messages. (link:https://issues.redhat.com/browse/LOG-3074[LOG-3074])
```
Comment (rolfedh) [inline modules/cluster-logging-rn-5.4.9.adoc:9]:
```suggestion
* Before this update, the Fluentd collector warned of unused configuration parameters. This update removes those configuration parameters and their warning messages. (link:https://issues.redhat.com/browse/LOG-3074[LOG-3074])
```

Comment (rolfedh) [reply modules/cluster-logging-rn-5.4.9.adoc:9]: ISG "Do not use the subjunctive mood in technical documentation."

**Review by rolfedh — COMMENTED** [review]
Nice work. Noted some issues and nits.

File: logging/cluster-logging-release-notes.adoc (hunk 2 above)
Comment (rolfedh) [reply logging/cluster-logging-release-notes.adoc:393]: ISG: may verb
Use only to indicate permission. Otherwise, use "can" to indicate ability, or use "might" to indicate possibility.
"May" is overused in technical information. In most cases, use "can" or "might" instead.

**Comment by libander** [comment]
> Nice work. Noted some issues and nits. 

Thank you, I'll follow up on those at a later time.
